tests/registry-command/test-ecr.sh

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env bash
+#!/usr/bin/env sh
```

Comment:
For some reason I thought this trap required bash, but if that's not the case, then there's no need for `env` either. (I only used `env` since `bash` as a matter of habit since it doesn't always get installed in the same location)
```suggestion
#!/bin/sh
```

Reply:
sorry, that was just a mistake after I rebased, I reverted this change for now.